start.sh

```diff
@@ -219,7 +202,14 @@ setupCouchbase() {
     N1QLAPI=$(getIpPort couchbase 8093)
     echo
     echo 'Creating Couchbase buckets'
-    removeBucket benchmark
+    while true; do
+        echo -n '.'
+        curl -sf -u ${COUCHBASE_USER}:${COUCHBASE_PASS} \
```

Comment:
This section appears in progress, but my question is if you're planning to move the bucket and index creation to the Touchbase `preStart` script.